OpenEdXMobile/src/main/java/org/edx/mobile/view/SplashActivity.java

```diff
@@ -48,6 +57,27 @@ protected void onCreate(Bundle savedInstanceState) {
         } else {
             environment.getRouter().showLaunchScreen(SplashActivity.this);
         }
+        initRemoteConfigs();
+    }
+
+    private void initRemoteConfigs() {
+        if (config.getFirebaseConfig().isEnabled()) {
+            FirebaseRemoteConfig firebaseRemoteConfig = FirebaseRemoteConfig.getInstance();
+            FirebaseRemoteConfigSettings configSettings = new FirebaseRemoteConfigSettings.Builder()
+                    .setMinimumFetchIntervalInSeconds(3600)
+                    .build();
+            firebaseRemoteConfig.setConfigSettingsAsync(configSettings);
+            firebaseRemoteConfig.setDefaults(R.xml.remote_config_defaults);
+            firebaseRemoteConfig.fetchAndActivate().addOnCompleteListener(task -> {
```

Comment:
How often does this run?

Reply:
> How often does this run?

`firebaseRemoteConfig.fetchAndActivate()` statement send the request to get the remote config and call `addOnCompleteListener` method after completing the fetch task.

Reply:
I see. Does this run once when the app launches? Or periodically while it's in the foreground? When does it re-run?

I would like the event to fire once when the user opens the app.

Reply:
> Does this run once when the app launches?
Yes, it runs only once when ever user start the app and fire the event.